flatpak.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file does nothing anyway as it has the wrong filename.

See https://github.com/flathub/flathub/pull/2420#issuecomment-879684329 on how to resolve the aarch64 build issue.